### PR TITLE
Management command for optout sync

### PR DESCRIPTION
**Pull request by imsickofmaps**
CSV uploader and celery task now

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@smn ready for review

File: subscription/views.py [hunk 1]
```diff
@@ -35,3 +35,32 @@ def uploader(request, page_name):
         context.update(csrf(request))
         return render_to_response("custom_admin/upload.html", context,
                                   context_instance=RequestContext(request))
+
+
+@staff_member_required
+def outout_uploader(request, page_name):
```
Comment (smn) [inline subscription/views.py]: typo?

File: subscription/forms.py [hunk 2]
```diff
@@ -9,3 +9,10 @@ class CSVUploader(forms.Form):
     def save(self):
         csv_data = StringIO(self.cleaned_data["csv"].read())
         ingest_csv.delay(csv_data, self.cleaned_data["message_set"])
+
+
+class OptOutCSVUploader(forms.Form):
+    csv = forms.FileField()
+    def save(self):
+        csv_data = StringIO(self.cleaned_data["csv"].read())
+        ingest_opt_opts_csv.delay(csv_data)
```
Comment (smn) [inline subscription/forms.py:18]: Does a `StringIO()` object survive via serialisation & celery? Wouldn't it be more straight forward to just send through the string you get back from `.read()` ?

Comment (imsickofmaps) [inline subscription/forms.py:18]: It has until now with the other task this is following the pattern of

Comment (smn) [inline subscription/forms.py:18]: Surprising but ok if you say so.

**Comment by smn** [comment]
:+1:

**Pull request merged**